src/components/Company/Offers/Manage/CompanyOffersManagementWidget.js

```diff
@@ -51,16 +52,17 @@ const RowActions = () => (
 
 
 const CompanyOffersManagementWidget = () => {
+    const { data, isLoggedIn } = useSession();
     const [offers, setOffers] = useState(null);
     const [isLoading, setIsLoading] = useState(true);
     const [error, setError] = useState(null);
 
     useEffect(() => {
-        fetchCompanyOffers().then((offers) => {
+        if (isLoggedIn) fetchCompanyOffers(data.company._id).then((offers) => {
             /* TODO: SHOULD NOT RUN WHEN COMPONENT IS UNMOUNTED: CANCEL PROMISE */
```

Comment:
When will this be done? Maybe we can open an issue to reference this.

Reply:
That is done in #96, kind of my bad for letting that rest for half a year :grimacing: 

In my defense, it didn't have that much priority ahaha